dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/Dataset.java

```diff
@@ -227,16 +227,17 @@ public void setGuestbook(Guestbook guestbook) {
     }
 
     /**
-     * Timer used mainly for OAI exporters to determinate whether metadata was changed or not,
-     *  since license in exporters could be changed based on guestbook.
-     *
+     * Timer used for OAI exporters. Holds metadata accessibility last change time.
+     * Example: gestbook and embargo operations can happen after dataset was published that
+     * affect metadata visibility for harvesting clients, without directly changing
+     * metadata itself.
```

Comment:
`... guestbook assigning and embargo date change can happen after dataset was published and without generating new dataset version. This operations can affect the final result of some exporters which should be reflected in OAI incremental imports.`?
Mind typo in `gestbook`. I don't know if we could tell that it concerns only `visibility` or `accessibility` so I would try to avoid that statements